bufferserver/src/test/java/com/datatorrent/bufferserver/client/SubscriberTest.java

```diff
@@ -66,16 +66,16 @@ public static void setupServerAndClients() throws Exception
     eventloopServer.start();
     eventloopClient.start();
 
-    instance = new Server(0, 64, 2);
-    address = instance.run(eventloopServer);
+    instance = new Server(eventloopServer, 0, 64, 2);
+    address = instance.run();
```

Comment:
`eventloop` is a member variable and it is more common to have such variables initialized in a constructor. I also wanted to change it to final (done now).